model/patch/github.go

```diff
@@ -77,6 +83,9 @@ type githubIntent struct {
 
 	// IntentType indicates the type of the patch intent, e.g. GithubIntentType
 	IntentType string `bson:"intent_type"`
+
+	// CalledBy indicates what created the intent
```

Comment:
"whether the intent was created automatically by Evergreen or manually" or something like that is more accurate, to distinguish it from IntentType